packages/protocol/test/stability/reserve.ts

```diff
@@ -283,6 +286,115 @@ contract('Reserve', (accounts: string[]) => {
     })
   })
 
+  describe('#addExchangeSpender(exchangeAddress)', () => {
+    it('only allows owner', async () => {
+      await assertRevert(reserve.addExchangeSpender(nonOwner, { from: nonOwner }))
+    })
+
+    it('should emit addExchangeSpender event on add', async () => {
+      const resp = await reserve.addExchangeSpender(exchangeSpenderAddress)
+      const log = resp.logs[0]
+      assert.equal(resp.logs.length, 1)
+      assertLogMatches2(log, {
+        event: 'ExchangeSpenderAdded',
+        args: {
+          exchangeSpender: exchangeSpenderAddress,
+        },
+      })
+    })
+
+    it('has the right list of exchange spenders after addition', async () => {
+      await reserve.addExchangeSpender(exchangeAddress)
+      await reserve.addExchangeSpender(accounts[1])
+      const spenders = await reserve.getExchangeSpenders()
+      assert.equal(spenders.length, 2)
```

Comment:
You should be able to check lists with the same contents in a single assert statement.